### PR TITLE
Make image-builder always pull latest eks-d artifacts

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Make image-builder cli decouple from a specific tag of build tooling repo. This will enable image-builder to pull latest eks-d releases.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/aws/image-builder/builder/utils.go [hunk 1]
```diff
@@ -7,9 +7,9 @@ import (
 	"os/exec"
 )
 
-func cloneRepo(cloneUrl, destination, tag string) error {
+func cloneRepo(cloneUrl, destination string) error {
 	log.Print("Cloning eks-anywhere-build-tooling...")
-	cloneRepoCommandSequence := fmt.Sprintf("git clone %s %s; cd %s; git checkout %s", cloneUrl, destination, destination, tag)
+	cloneRepoCommandSequence := fmt.Sprintf("git clone %s %s", cloneUrl, destination)
```
Comment (jaxesn) [inline projects/aws/image-builder/builder/utils.go:12]: one thing to keep in mind about this approach in general. in code build we clone from codecommit vs github. Im not saying its a huge deal that we are now cloning this from github, but its something to call out.

Comment (vignesh-goutham) [reply projects/aws/image-builder/builder/utils.go:12]: yep, Im working on that actually.

Comment (jaxesn) [reply projects/aws/image-builder/builder/utils.go:12]: Also this is not exactly right anyway... this wouldnt work for release branches?

**Pull request merged**